platform-sdk/docs/proposals/TSS-Ledger-Id/TSS-Ledger-Id.md

```diff
@@ -0,0 +1,826 @@
+    * This requires modification of the HAPI transactions that manage the address book.
+
+Impacts to DevOps Team
+
+* Each consensus node will need a new long term EC key in addition to the existing RSA key.
+    * EC Key generation will have to happen before a node can join the network.
+* A new node added to an existing network will need to be given a state from an existing node after the network has
+  adopted the address book containing the new node.
+
+Implications Of Completion
+
+* The consensus nodes will be able to create ledger signatures for use in block proofs.
+* TSS-Block-Signing proposal becomes unblocked for development.
+
+### Requirements
+
+TSS Core Requirements
+
+| Requirement ID | Requirement Description                                                                                                      |
+|----------------|------------------------------------------------------------------------------------------------------------------------------|
+| TSS-001        | The ledger MUST have a public/private signing key pair that does not change when the consensus roster changes.               |
+| TSS-002        | The ledger private key MUST be a secret that nobody knows.                                                                   |
+| TSS-003        | The `ledger id` MUST be a public key that everyone knows and able to verify signatures from the ledger private key.          |
+| TSS-004        | The ledger signature for a message MUST be produced through a threshold number of consensus nodes signing the message.       |
+| TSS-005        | There MUST be a way to bootstrap an existing network's consensus roster with to give the ledger a public/private signing key |
+| TSS-006        | There MUST be a way for a new network to generate a public/private signing key for the ledger.                               |
+| TSS-007        | There MUST be a way to rotate the ledger's public/private signing key.                                                       |
+| TSS-008        | The ledger signature SHOULD be verifiable by an EVM smart contract without excessive cost or time.                           |
+| TSS-009        | The TSS implementation SHOULD be able to switch elliptic curves.                                                             |
+| TSS-010        | The TSS algorithm SHOULD be able to model consensus weight with high precision.                                              |
+| TSS-011        | Minimize the number of steps a Node Operator needs to perform.                                                               |
+
+Block Signing Requirements
+
+| Requirement ID | Requirement Description                               |
+|----------------|-------------------------------------------------------|
+| TSSBS-001      | The initial block in the block stream MUST be signed. |
+| TSSBS-002      | On a new network, the initial block MUST be block 0   |
+
+### Design Decisions
+
+#### New Address Book Life-Cycle
```

Comment:
I think this entire section belongs in the TSS-Roster proposal. @kfa-aguda do you agree?

Reply:
While the TSS-Roster does the bulk of the changes, the `TSS-Ledger-ID` proposal has to modify the life-cycle from where the TSS-Roster left off.  I am recounting the TSS-Roster life-cycle so I have a baseline to change.  I can leave it off if you like and tell the reader to go read the TSS-Roster proposal before this one.

Reply:
I've removed the section that presents the TSS-Roster life-cycle.